### DISCUSSION TITLE
[Release Vote] Vote for Release v0.5.1-rc1

### DISCUSSION THREAD

**Discussion by wangtaohz** [apache/amoro#2025] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Amoro 0.5.1 release.

The commit ID is https://github.com/NetEase/amoro/commit/8f3b304aa3e00b6220b17c99e4f6f33762857e63

This corresponds to the tag: v0.5.1-rc1
https://github.com/NetEase/amoro/commits/v0.5.1-rc1
https://github.com/NetEase/amoro/tree/v0.5.1-rc1
Please check out, build, and test.

Please vote in the next 72 hours and add your test steps when voting.

**Comment by wangtaohz** [comment]
+1.

I have built and deployed AMS locally with jdk8, hadoop3, flink-1.15.

Tested the following features:

- using derby as sysdb
- create the optimizer group (local container) and start a local optimizer
- create internal catalog (both Mixed Iceberg and Iceberg)
- spark insert overwrite/flink cdc (both Mixed Iceberg and Iceberg)
- table optimizing and result

**Comment by baiyangtx** [comment]
-1:  There is a problem in MixedIceberg within ExternalCatalog and Kerberos

- https://github.com/NetEase/amoro/issues/2029

**Reply by wangtaohz** [reply]
Thank you for your report, the issue has been fixed.

**Comment by wangtaohz** [comment]
Tag 0.5.1-rc2 has been created. https://github.com/NetEase/amoro/discussions/2048
